### PR TITLE
Update User Guide to v1.2 and adjust output formatting

### LINKED ISSUE #70: Update UG
Update UG to fit 1.3 milestone

### LINKED ISSUE #90: Fix print format issues for Recipe and Ingredient
- Recipe currently prints with numbers for steps under view but not for the output of each command. 
- The sequence is also unintuitive (i.e. name, ingredients, steps then description) for `view recipe`
- Some style issues overall for the outputs of commands from both Recipe and Ingredient.

### PR DESCRIPTION
**Pull request by NayLin-H99**
Fixes #70.
Fixes #90.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -61,7 +83,48 @@ Format: `delete ingredient <index>`
...
+Example: 
+1. `edit ingredient 1 -d juicy`
+<br />Expected Output:<br />`Edited Ingredient: Watermelon; Quantity: 1; Expiry Date: 20-10-2021; Description: juicy; Tags: [expiring][melon][water]`
+
+### Clear Ingredients:
+`clear`
+
+Clear all the ingredients from the inventory of ingredients
+
+Format: `clear`
```
Comment (pattan9955) [inline docs/UserGuide.md:104]: I think it should be 'clear ingredient' not clear, might have to update this

File: docs/UserGuide.md [hunk 2]
```diff
@@ -88,12 +152,29 @@ Format: `delete recipe <index>`
...
 -----
 # Command Summary
+
 Action | Format
 --------|------------------
-**Add Ingredient** | `add ingredient -n <name> -t <type> -q <quantity> [-d <description>] -e <expiry date>`
+**Add Ingredient** | `add ingredient -n <name> -q <quantity>[<units>] [-d <description>] -e <expiry date>`
 **Delete Ingredient** | `delete ingredient <index>`
+**Edit Ingredient** | `edit ingredient <index> -<field flag> <new data>`
+**Clear Ingredient** | `clear`
```
Comment (pattan9955) [inline docs/UserGuide.md:175]: Same as previous, need to update to "clear ingredient" I believe

File: docs/UserGuide.md (hunk 1 above)
Comment (NayLin-H99) [reply docs/UserGuide.md:104]: This is based on our current implementation.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -88,12 +152,29 @@ Format: `delete recipe <index>`
...
+**Find Ingredient** | `find ingredient <keyword>`
```
[the remark this answers is not in the document]
Comment (NayLin-H99) [reply docs/UserGuide.md]: Adjusted after merging #79
Thanks for letting me know.

**Review by alissayarmantho — COMMENTED** [review]
LGTM

**Pull request merged**